aptos-move/aptos-vm/src/aptos_vm.rs

```diff
@@ -2415,12 +2415,19 @@ impl AptosVM {
                 .as_move_value(),
         ];
 
+        let mut function_name = BLOCK_PROLOGUE_EXT;
+
+        if self.features().is_enabled(FeatureFlag::NEW_FLAG) {
+            args.insert(0, MoveValue::Signer(AccountAddress::ONE));
```

Comment:
Giving a framework `signer` to the prologue that wasn't there before?

Security issues?

Reply:
today: call `block_prologue_ext` without framework signer.
After the feature is enabled: call `block_prologue_ext_v2` with the signer.

should be fine?

Reply:
in principle fine for now

but in the future, `block_prologue_ext_v2` implementations could misues the `signer`...